### PR TITLE
child_repository_refactoring

### PR DESCRIPTION
**Pull request by AliakseiRabushka**
child repository refactoring

### CODE REVIEW

**Comment by AliakseiRabushka** [comment]
> рефакторинг дочернего репозитория

code formatting in a project

**Review by AliakseiRabushka — COMMENTED** [review]
conflict settled

File: src/main/java/com/example/kindergarten/service/ChildServiceImpl.java [hunk 1]
```diff
@@ -9,7 +9,10 @@
 import com.example.kindergarten.repository.ChildRepository;
 import com.example.kindergarten.repository.UserRepository;
 
+child_repository_refactoring
```
Comment (belnarto) [inline src/main/java/com/example/kindergarten/service/ChildServiceImpl.java]: ??

File: src/main/java/com/example/kindergarten/service/ChildServiceImpl.java [hunk 2]
```diff
@@ -109,7 +112,8 @@ public List<ChildDto> searchByCategory(String category) {
 
     @Override
     public List<ChildDto> searchByName(String name) {
-        return childRepository.findByPartName(name).stream()
+ child_repository_refactoring
```
Comment (belnarto) [inline src/main/java/com/example/kindergarten/service/ChildServiceImpl.java]: ??

File: src/main/java/com/example/kindergarten/service/ChildServiceImpl.java [hunk 3]
```diff
@@ -121,8 +125,11 @@ public List<ChildDto> searchByAge(int minAge, int maxAge) {
                 .collect(Collectors.toList());
     }
 
+
+
```
Comment (belnarto) [inline src/main/java/com/example/kindergarten/service/ChildServiceImpl.java]: why do we need extra empty lines here?

File: src/main/java/com/example/kindergarten/service/ChildServiceImpl.java [hunk 4]
```diff
@@ -121,8 +125,11 @@ public List<ChildDto> searchByAge(int minAge, int maxAge) {
...
     private ChildEntity setCommonFieldsFromDtoToEntity(ChildDto childDto) {
         return ChildEntity.builder()
+
```
Comment (belnarto) [inline src/main/java/com/example/kindergarten/service/ChildServiceImpl.java]: ????

**Review by belnarto — CHANGES_REQUESTED** [review]

**Review by belnarto — APPROVED** [review]

**Pull request merged**